README.md

```diff
@@ -20,21 +20,21 @@ This is achieved by creating YAML files that contain a pre-determined data schem
 
 Role: [cisco.nac_dc_vxlan.validate](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/validate/README.md)
 
-The validate role function is to ensure that the data model is correct, and that the data model is going to be able to be processed by the subsequent roles. The validate role is going to read all the files in the `host_vars` directory and create a single data model in memory for execution.
+The validate role has the function to ensure that the data model is correct, and that the data model is going to be able to be processed by the subsequent roles. The validate role is going to read all the files in the `host_vars` directory that is named after the inventory group defined in your inventory file and create a single data model in memory for execution.
 
-As part of the VXLAN as Code service from Cisco, you will also be able to utilize the semantic validation to make sure that the data model matches the intended expected values. This is a powerful feature that allows you to ensure that the data model is correct before it is deployed to the network. Also, part of the validate role is the ability to create rules that can be used to avoid operators from making specific configurations that are not allowed in the network. These can be as simple as ensuring naming convention to more complex rules for interconnectivity that would need to be avoided. These would be coded in python and can be constructed as part of the Services as Code offer.
+As part of the VXLAN as Code service offer from Cisco, you will also be able to utilize the semantic validation to make sure that the data model matches the intended expected values. This is a powerful feature that allows you to ensure that the data model is correct before it is deployed to the network. Also, part of the validate role is the ability to create rules that can be used to avoid operators from making specific configurations that are not allowed in the network. These can be as simple as ensuring naming convention to more complex rules for interconnectivity that would need to be avoided. These would be coded in python and can be constructed as part of the Services as Code offer.
 
 ### Create role
 
 Role: [cisco.nac_dc_vxlan.dtc.create](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/dtc/create/README.md)
 
-This role is going to create all the templates and variable parameters that are going to be used in the deployment of the VXLAN fabric. This role converts the data model into the proper templates that are required by the Ansible module to be able to communicate with the NDFC controller.
+This role is going to use a templates and variable parameters from the data model to create and stage all the configuration for your VXLAN fabric. This role uses the data model in the proper templates to render the expected data that is required by the NDFC Ansible modules.
```

Comment:
There are two steps. First, this role creates the configuration from the templates and variables. (Apparently by including nac_dc_vxlan/roles/dtc/common/tasks/main.yml, although I don't get how it is included.) Second, this roles applies the configuration to the NDFC, thereby creating the fabric on the NDFC. (By including nac_dc_vxlan/roles/dtc/create/tasks/main.yml.)

Reply:
Hi @marehler, correct, but I don't think we need explicitly expand on all the dependencies and details under the covers as that's the point in the collection in taking that burden off the consumer. Essentially, one should not need to dive into the role. 

Also agree with your second sentence and that is what is meant by staging the configuration in NDFC. Create stages and does not push to the switches (except in a few one off scenarios to ensure certain things are in a working order for the collection to run). 

Let me know if that doesn't make sense or if we're crossing wires.

Reply:
Maybe the term "stage" is a bit unclear. Besides it says "stage", not "stage in NDFC". Therefore I thought it meant just creating and preparing the configuration in Ansible but not pushing them yet to NDFC. How about rephrasing the first sentence to "[...] to create the configuration in NFDC"? Then it should be clear.

Reply:
The documentation in our example repo is clearer so I propose we use that here as well.

Reply:
Agree, that is clearer.